modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -584,6 +584,9 @@ private ObjectDefinition _toObjectDefinition(
 				pluralLabel = LocalizedMapUtil.getLanguageIdMap(
 					objectDefinition.getPluralLabelMap());
 				portlet = objectDefinition.getPortlet();
+				restContextPath =
+					(objectDefinition.isSystem() ? "/o/" : "/o") +
```

Comment:
Why did you prefer to add "/o/" or "/o" in this layer? I was thinking why could we not do it in ObjectDefinitionImpl.java > getRESTContextPath(); ?
Let me know your opinion about it :+1: